lib/tasks/turbo_tasks.rake

```diff
@@ -1,24 +1,29 @@
-def run_turbo_install_template(path) system "#{RbConfig.ruby} ./bin/rails app:template LOCATION=#{File.expand_path("../install/#{path}.rb",  __dir__)}" end
+def run_turbo_install_template(path, nspace = nil)
+  system "#{RbConfig.ruby} #{Rails.root}/bin/rails #{nspace}app:template LOCATION=#{File.expand_path("../install/#{path}.rb",  __dir__)}"
+end
 
 namespace :turbo do
   desc "Install Turbo into the app"
-  task :install do
+  task :install do |task|
+    nspace = task.name.split(/turbo:install/).first
```

Comment:
This is ultimately the approach that I took to determine whether the rake task invocations should include the app namespace. An alternative approach I was considering is something like:

```ruby
nspace = nil
nspace = "app:" if defined?(ENGINE_ROOT) 
```
I came across `ENGINE_ROOT` from one of the [rails source files](https://github.com/rails/rails/blob/main/railties/lib/rails/tasks/engine.rake), but I don't know enough of rails to be confident that will always work.